### DISCUSSION TITLE
Hexo-给博客提提速！ | Zi-Zi's Journey

### DISCUSSION THREAD

**Discussion by giscus** [GZ-Metal-Cell/GZ-Metal-Cell.github.io#301] [Announcements]
# Hexo-给博客提提速！ | Zi-Zi's Journey

promefire😭你太强了😭

https://gz-metal-cell.github.io/2024/01/10/Hexo-%E7%BB%99%E5%8D%9A%E5%AE%A2%E6%8F%90%E6%8F%90%E9%80%9F%EF%BC%81/

**Comment by promefire** [comment]
Metal-Cell 太强了！！ 评论好使了

**Reply by GZ-Metal-Cell** [reply]
有请 **pro 神**莅临寒舍😭
